File: SDKs/beezup.com/2.0/java/lib/src/main/java/openapisdk/models/shared/CompressionFormatStrategyEnum.java
package openapisdk.models.shared;


public enum CompressionFormatStrategyEnum {
    NONE("None"),
    ZIP("Zip"),
    TAR_GZ("TarGz"),
    GZIP("Gzip"),
    BZIP2("Bzip2"),
    RAR("Rar");

    public final String value;

    private CompressionFormatStrategyEnum(String value) {
        this.value = value;
    }
}

File: SDKs/beezup.com/2.0/python/src/sdk/models/shared/compressionformatstrategy_enum.py
import dataclasses
from enum import Enum

class CompressionFormatStrategyEnum(str, Enum):
    NONE = "None"
    ZIP = "Zip"
    TAR_GZ = "TarGz"
    GZIP = "Gzip"
    BZIP2 = "Bzip2"
    RAR = "Rar"

